FWCore/ParameterSet/python/Config.py

```diff
@@ -2842,7 +2941,12 @@ def testSubProcess(self):
 process.addSubProcess(cms.SubProcess(process = childProcess, SelectEvents = cms.untracked.PSet(
 ), outputCommands = cms.untracked.vstring()))"""
             equalD = equalD.replace("parentProcess","parentProcess"+str(hash(process.subProcesses_()[0])))
-            self.assertEqual(_lineDiff(d,Process('Parent').dumpPython()+Process('Child').dumpPython()),equalD)
+            messageLoggerSplit = 'process.MessageLogger = cms.Service'
```

Comment:
How about a comment explaining why this was done?

Reply:
Added comment.